src/Core/ICredentials.cs

```diff
@@ -21,7 +21,7 @@
 using System;
 using SonarQube.Client.Models;
 
-namespace SonarLint.VisualStudio.Integration.Persistence
+namespace SonarLint.VisualStudio.Core
```

Comment:
Why `Core` rather than `Core.Binding`?

Reply:
I thought it was credentials in general. Moved to Binding.